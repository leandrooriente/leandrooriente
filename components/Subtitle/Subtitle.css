.title {
  font-family: var(--headline-font-family);
  font-size: 28px;
  font-weight: normal;
  margin: 0 0 20px;
}