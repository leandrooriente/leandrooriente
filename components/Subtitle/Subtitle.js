import React from 'react';
import PropTypes from 'prop-types';
import css from './Subtitle.css';

const Subtitle = ({ children }) => (
  <h3 className={css.title}>
    { children }
  </h3>
);

Subtitle.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Subtitle;
