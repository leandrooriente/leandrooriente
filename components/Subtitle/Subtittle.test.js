import React from 'react';
import { mount } from 'enzyme';
import Subtitle from './Subtitle';

describe('<Subtitle />', () => {
  it('should render <Subtitle /> component', () => {
    const wrapper = mount(
      <Subtitle>
        Test
      </Subtitle>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
