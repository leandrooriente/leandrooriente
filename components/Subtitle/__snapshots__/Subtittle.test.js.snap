// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Subtitle /> should render <Subtitle /> component 1`] = `
<Subtitle>
  <h3
    className="title"
  >
    Test
  </h3>
</Subtitle>
`;
